lib/wings/hydra/pcdm/models/concerns/pcdm_valkyrie_behavior.rb

```diff
@@ -43,20 +43,26 @@ def parent_collection_ids(valkyrie: false)
 
       ##
       # @return [Enumerable<ActiveFedora::Base> | Enumerable<Valkyrie::Resource>] an enumerable over the child collections
+      # TODO: All collections will be natrually ordered when the application is valkyrie-native, aliasing ordered_collection*
```

Comment:
i'm not following the meaning of this comment. could we rephrase?

it might be good to use `@todo`, for to do items as well. https://www.rubydoc.info/gems/yard/file/docs/Tags.md#todo